<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trouble_code</class>
 <widget class="QDialog" name="trouble_code">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fault_info_laber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(164, 0, 0)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;故障说明&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="fault_code_laber">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(30, 191, 238)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;故障代码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_fault">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 255, 127, 255), stop:0.113636 rgba(170, 255, 255, 255), stop:0.238636 rgba(34, 166, 248, 255), stop:0.494318 rgba(0, 255, 207, 255), stop:0.767045 rgba(34, 166, 248, 255), stop:0.892045 rgba(170, 255, 255, 255), stop:1 rgba(85, 255, 127, 255))</string>
   </property>
   <property name="text">
    <string>清除故障码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>650</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(9, 41, 4, 255), stop:0.085 rgba(34, 166, 248, 255), stop:0.261364 rgba(170, 255, 255, 255), stop:0.267045 rgba(170, 255, 255, 255), stop:0.6875 rgba(170, 255, 255, 255), stop:0.885 rgba(34, 166, 248, 255), stop:1 rgba(0, 0, 0, 255));</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QListView" name="fault_desc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>301</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="fault_code_list">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>301</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
